packages/main/src/plugin/index.ts

```diff
@@ -445,6 +447,9 @@ export class PluginSystem {
       await trayIconColor.init();
     }
 
+    // Add all notifications to notification registry
+    notifications.forEach(notification => notificationRegistry.addNotification(notification));
+
```

Comment:
```suggestion
notifications.length = 0;
Object.freeze(notifications);
```

probably need to make notifications immutable after we've used the array content